### ISSUE TITLE
Configurations for the new color/style used in metadata line and warning for the scrollable list view

### ISSUE THREAD

**Issue by daxian-dbw** [PowerShell/PSReadLine#3584]
### Prerequisites

- [X] Write a descriptive title.

### Description of the new feature/enhancement

This issue is to follow up on https://github.com/PowerShell/PSReadLine/pull/3583#discussion_r1109181920, the discussion about what to make configurable about the new color and styles used in the metadata line and warning for the scrollable list view.

About the new colors and styles, there are a few of them:
- un-highlighted texts in the metadata line
   - _currently_, the un-highlighted text uses the default inline view color/style -- this is fixed as of today.
- highlighted text in the metadata line
   - _currently_, the highlighted text uses the `ListPredictionColor` plus `italics` -- so, it partially configurable.
- warning message
   - _currently_, the warning message uses the `ListPredictionColor` plus `italics`.

I'm not sure about what to expose for configuration yet. It doesn't seem needed to expose them all in my opinion. The following are my thoughts at the moment:
1. always apply `italics` to the text rendered in the metadata line, and use the `ListPredictionColor` as the user-configurable highlighting color/style.
2. maybe expose configuration for un-highlighted text color/style, but I'm not super sure if it's really needed.
3. warning message will be uncommon for users to see because it will happen usually only when the terminal height is very small (<5). So, exposing a configuration for it seems overkill to me.

I don't have a clear idea what/how to make configurable about the new color/styles at the moment, so I prefer to wait and collect some feedback about this, which hopefully will give us a clearer idea.

**Comment by StevenBucher98** [comment]
I agree with your three points, the only other customizable option that I think may be good to add is a way to disable the metadata line entirely. However I agree I think waiting for feedback would be best way to drive direction.

**Comment by daxian-dbw** [comment]
With the list view being scrollable, it may not be that useful to allow a user to disable the metadata line entirely. It would be hard for one to know where they are and what are available without the metadata line.

However, if a user only uses the history source, then there will be no problem without the metadata line.

**Comment by mezdelex** [comment]
In my opinion it should be customizable as in allowing the user to disable it completely or to disable italics at least. Italics have always been quite a controversial topic among many editors/plugins so I would add those options for the sake of customizability...

**Comment by joaoopereira** [comment]
IMHO, the user should be able to disable this new metadata line

**Comment by johan-lindqvist** [comment]
Seeing as the old PowerShell (<7.4) is now deprecated I would really like to be able to disable the metadata view and can't really see a reason for why it shouldn't be configurable. Let the user decide what to do with it.
